src/js/common/schemaform/review/ArrayField.jsx

```diff
@@ -201,11 +208,10 @@ class ArrayField extends React.Component {
                 <div className="row small-collapse">
                   <SchemaForm
                       reviewMode
-                      pageData={item}
-                      schema={this.getItemSchema(index)}
+                      data={item}
+                      schema={itemSchema}
                       uiSchema={arrayPageConfig.uiSchema}
-                      title={pageTitle}
-                      hideTitle
+                      title={itemSchema.title || ''}
```

Comment:
I think we'd want to default to `pageTitle` if there's no title in the item schema?